packages/url_launcher/url_launcher/example/ios/RunnerUITests/URLLauncherUITests.m

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import XCTest;
+@import os.log;
+
+@interface URLLauncherUITests : XCTestCase
+@property(nonatomic, strong) XCUIApplication* app;
+@end
+
+@implementation URLLauncherUITests
+
+- (void)setUp {
+  self.continueAfterFailure = NO;
+
+  self.app = [[XCUIApplication alloc] init];
+  [self.app launch];
+}
+
+- (void)testLaunch {
+  XCUIApplication* app = self.app;
+
+  NSArray<NSString*>* buttonNames = @[@"Launch in app", @"Launch in app(JavaScript ON)", @"Launch in app(DOM storage ON)", @"Launch a universal link in a native app, fallback to Safari.(Youtube)"];
```

Comment:
Interesting, I wonder if it's actually intentional that the last one falls back to SafariVC...